recipes/libusb/all/conanfile.py

```diff
@@ -35,9 +35,11 @@ def configure(self):
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
+        if self.settings.os == "Android":
+            self.options.enable_udev = False
```

Comment:
it should be in config_options(), not in configure(), otherwise consumers can't enable this option on Android.